### PR TITLE
Add `messages-square` icon

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
Nice!

I actually have a few drafts for this which are pretty similar (heck, the smaller speech bubble is literally identical), but I actually prefer yours, as it's more consistent with the existing `message-square` icon:
image

**Review by karsa-mistmere — APPROVED** [review]

File: icons/messages-square.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "comment",
+    "chat",
+    "conversation"
```
Comment (karsa-mistmere) [inline icons/messages-square.json:6]:
```suggestion
    "conversation",
    "copy",
    "multiple"
```

**Comment by danielbayley** [comment]
> Nice!
> 
> I actually have a few drafts for this which are pretty similar (heck, the smaller speech bubble is almost identical), but I actually prefer yours, as it's more consistent with the existing `message-square` icon: image

I do like the last 2 though… Maybe add one as an alternate?

**Comment by ericfennis** [comment]
I prefer the last one. The best use case for this one is I think a "conversation" which is more two message balloons on each site talking to each other. The current proposal is I think a bit off

File: icons/messages-square.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M14 9c0 1.1-.9 2-2 2H6l-4 4V5c0-1.1.9-2 2-2h8c1.1 0 2 .9 2 2v4z" />
+  <path d="M18 9h2c1.1 0 2 .9 2 2v10l-4-4h-6c-1.1 0-2-.9-2-2" />
```
Comment (ericfennis) [inline icons/messages-square.svg:13]: I think we could use a bit more of the canvas, the icons will look better on 100% scale. The message balloon looks a bit squished now.

Suggestion improvement (right): 
image


```suggestion
  <path d="M14 9a2 2 0 0 1-2 2H6l-4 4V4c0-1.1.9-2 2-2h8a2 2 0 0 1 2 2v5Z" />
  <path d="M18 9h2a2 2 0 0 1 2 2v11l-4-4h-6a2 2 0 0 1-2-2v-1" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]

**Review by ericfennis — APPROVED** [review]
Nice work!! LGTM

**Pull request merged**